src/main/java/org/unicode/cldr/surveydriver/App.java

```diff
@@ -0,0 +1,12 @@
+package org.unicode.cldr.surveydriver;
+
+/**
+ * Hello world!
+ *
+ */
+public class App {
+
+  public static void main(String[] args) {
+    System.out.println("Hello World! -- this is main");
+  }
+}
```

Comment:
I suppose we should get rid of App.java, assuming all the real code will be under src/test/java/...